source/maintainer-guide/installing/installing-from-the-installer.rst

```diff
@@ -3,3 +3,61 @@
 Installing from the installer
 #############################
 
+
+Overview
+--------
+
+A java installer package is available as one of the options to install GeoNetwork. 
+This article introduces the java installer package. Goal is to install GeoNetwork.
+GeoNetwork java installer runs on many common operating systems, such as Windows, macOS and Linux.
+The installer installs GeoNetwork in a basic setup using en embedded Jetty container.
+
+Before you Start
+----------------
+
+- Verify you have enough privileges to download and install software. 
+- System requirements: modern processor, 2GB of free memory, 1GB of disk space.
+- Verify if available, else install a java runtime environment (Oracle or OpenJDK) version 1.8. 
+
+To verify if java is available type 'java -version' on the command line.
+
```

Comment:
Need an installing java section, recommend "apt-get for linux", and adopt open jdk for linux and mac